pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -780,6 +780,12 @@ func (og *operationGenerator) checkForFailedMount(volumeToMount VolumeToMount, m
 func (og *operationGenerator) markDeviceErrorState(volumeToMount VolumeToMount, devicePath, deviceMountPath string, mountError error, actualStateOfWorld ActualStateOfWorldMounterUpdater) {
 	if volumetypes.IsOperationFinishedError(mountError) &&
 		actualStateOfWorld.GetDeviceMountState(volumeToMount.VolumeName) == DeviceMountUncertain {
+
+		if actualStateOfWorld.IsVolumeDeviceReconstructed(volumeToMount.VolumeName) {
```

Comment:
In [markVolumeErrorState](https://github.com/kubernetes/kubernetes/blob/f4240cbf925ffc64697d4eabf3c6145d2948e840/pkg/volume/util/operationexecutor/operation_generator.go#L808) we pass `volumeToMount.PodName` to `IsVolumeReconstructed`. Is it possible to use `volumeToMount.PodName` in `markDeviceErrorState` too?